Tests/helper.py

```diff
@@ -27,8 +27,7 @@
     class test_image_results:
         @staticmethod
         def upload(a, b):
-            a.show()
-            b.show()
```

Comment:
What is the reason for this change?

Reply:
If it is hit, the tests will hang infinitely. I do not see any good reason for a.show(), b.show(): we are interested in return value of the **upload** function, and return value here is None.

Reply:
Huh? This is only used if there is an environment variale `SHOW_ERRORS` set, which means the user is asking to be shown failed image comparisons. Removing this breaks that functionality. Also, the result of the upload function is irrelevant as there is a check in place to see if it is defined. If it isn't defined, it is never called anyway, so if this were to be removed, you would remove the whole if block.